shell/gpu/gpu_surface_gl.cc

```diff
@@ -285,7 +285,8 @@ std::unique_ptr<SurfaceFrame> GPUSurfaceGL::AcquireFrame(const SkISize& size) {
         return weak ? weak->PresentSurface(canvas) : false;
       };
 
-  return std::make_unique<SurfaceFrame>(surface, submit_callback);
+  return std::make_unique<SurfaceFrame>(
+      surface, !delegate_->UseOffscreenSurface(), submit_callback);
```

Comment:
delegate->UseOffscreenSurface should probably be renamed given its new purpose. This can be done when I remove all of the offscreen logic from gpu_surface_gl.

Reply:
Done in the next commit. It will now be delegate->SurfaceSupportsReadback().